From 4018fb71cfccd7cf00bbc4f75eab5e4dcd6b5f28 Mon Sep 17 00:00:00 2001
From: Radu Pirea <radu-nicolae.pirea@nxp.com>
Date: Fri, 20 Nov 2020 17:18:48 +0200
Subject: [PATCH 04/76] campps32v2: update xrdc documentation reference

Changed Chapter number to its title because might differ between
revisions.

Issue: ALB-5671

Signed-off-by: Radu Pirea <radu-nicolae.pirea@nxp.com>
---
 board/freescale/campps32v2/campps32v2.c | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/board/freescale/campps32v2/campps32v2.c b/board/freescale/campps32v2/campps32v2.c
index ee54f7691b..06725bb072 100644
--- a/board/freescale/campps32v2/campps32v2.c
+++ b/board/freescale/campps32v2/campps32v2.c
@@ -190,7 +190,9 @@ int board_phy_config(struct phy_device *phydev)
 
 void setup_xrdc(void)
 {
-	/* See S32V234 User Manual Rev 2, 11.4.21.
+	/* See S32V234 User Manual, chapter Extended Resource Domain Controller
+	 * (XRDC), section S32V234 specific MRC instance for SRAM controller
+	 * memory protection.
 	 * Let ISP, Camera, Decoder Pixel Interface and Encoder Bit Stream to
 	 * access the SRAM memory.
 	 */
@@ -199,7 +201,7 @@ void setup_xrdc(void)
 	writel(XRDC_ADDR_MIN, XRDC_MRGD_W0_16);
 	/* Write end of the memory region. */
 	writel(XRDC_ADDR_MAX, XRDC_MRGD_W1_16);
-	/* Write valid bit for the memory region memory region. */
+	/* Write valid bit for the memory region */
 	writel(XRDC_VALID, XRDC_MRGD_W3_16);
 
 	writel(XRDC_ADDR_MIN, XRDC_MRGD_W0_17);
-- 
2.17.1

